Project1/application.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+import time
+
+from flask import Flask, session,url_for,redirect
+from flask import Flask,render_template,request,flash
+from flask_session import Session
+from  datetime import datetime
+from details import *
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+app = Flask(__name__)
+app.secret_key = 'any random string'
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+engine = create_engine(os.getenv("DATABASE_URL"))
+Session(app)
+db.init_app(app)
+
+with app.app_context():
+	db.create_all()
+
+@app.route("/")
+def index0():
+	
+	return redirect(url_for("register"))
+
+  
+
+
+@app.route("/register",methods = ["GET","POST"])
+def register():
+    if request.method == "GET":
+        return render_template("Registration.html")
+    elif request.method == "POST":
+        name = request.form.get("fname")
+        email = request.form.get("Email")
+        pswd  = generate_password_hash(str(request.form.get("password")))
+
+        register = Registration(Email = email, Password = pswd,dt = time.ctime(time.time()))
```

Comment:
Why is the argument dt sent when you are using datetime.now() in the constructor.